torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -264,20 +266,46 @@ def convert_name(token):
             return s[1:-1]
         return s
 
+    functions: Dict[str, Dict[Intermediate, List[Op]]] = {}
+
+    def extend_dict_list(d1, d2):
+        for key, values in d2.items():
+            d1[key].extend(values)
+
     @v_args(inline=True)
-    class CalculateOps(Transformer):
+    class TransformOps(Transformer):
         def process_op(self, ret, op_name, fn_name, args, *rest):
-            ops[convert(ret)] = Op(
-                convert_name(op_name), convert_name(fn_name), convert(args)
+            return Op(
+                convert_name(op_name),
+                convert_name(fn_name),
+                convert(args),
+                convert(ret),
             )
 
-        def process_func(self, name, *rest):
-            nonlocal ops
+        def process_func(self, name, _args, *stmts):
```

Comment:
Is linter happy about unused params if the name starts with `_`? That's interesting, didn't know. What I've seen before is that `_` is used, but this doesn't work with many.

Reply:
I dont think we have an unused parameter linter, I just use this as my own convention